src/kuzzleCollectionMapping.js

```diff
@@ -13,13 +13,13 @@
  *  (currently handled by ElasticSearch), and your searches may suffer from below-average performances, depending on
  *  the amount of data you stored in a collection and the complexity of your database.
  *
- *  The KuzzleDataMapping object allow to get the current mapping of a data collection and to modify it if needed.
+ *  The CollectionMapping object allow to get the current mapping of a data collection and to modify it if needed.
  *
- * @param {object} kuzzleDataCollection - Instance of the inherited KuzzleDataCollection object
+ * @param {object} kuzzleDataCollection - Instance of the inherited Collection object
```

Comment:
(and then rename all instances of `kuzzleDataCollection` to `collection` within the method)